connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/ReplicationPolicy.java

```diff
@@ -52,9 +52,39 @@ default String originalTopic(String topic) {
         }
     }
 
+    /** Returns heartbeats topic name.*/
+    default String heartbeatsTopic() {
+        return "heartbeats";
+    }
+
+    /** Returns the offset-syncs topic for given cluster alias. */
+    default String offsetSyncTopic(String clusterAlias) {
+        return "mm2-offset-syncs." + clusterAlias + ".internal";
+    }
+
+    /** Returns the name checkpoint topic for given cluster alias. */
+    default String checkpointTopic(String clusterAlias) {
+        return clusterAlias + ".checkpoint.internal";
```

Comment:
This topic is currently named `.checkpoints.internal`, see https://github.com/apache/kafka/blob/trunk/connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/MirrorClientConfig.java#L60. We need to keep the current name

Reply:
Can we also rationalize the different names. Ig uess this method should be `checkpointsTopic`.

Reply:
> This topic is currently named .checkpoints.internal, see https://github.com/apache/kafka/blob/trunk/connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/MirrorClientConfig.java#L60. We need to keep the current name

Good spot

> Can we also rationalize the different names. Ig uess this method should be checkpointsTopic.

make sense, will update the KIP as well to reflect this.